Cargo.lock

```diff
@@ -2893,6 +2893,7 @@ name = "rustc_data_structures"
 version = "0.0.0"
 dependencies = [
  "cfg-if 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
+ "crossbeam-utils 0.6.5 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I would prefer to not use `crossbream-utils` until such time that it has a more reasonable policy towards soundness and does not include both `get_mut` and `compare_exchange`. Since it seems that we want `get_mut` but not `compare_exchange` it seems preferable to make an in-house copy of `AtomicCell` for now.